core/engine/src/builtins/set/tests.rs

```diff
@@ -358,14 +358,14 @@ fn symmetric_difference_same_set() {
             let setA = new Set(["JavaScript", "HTML", "CSS"]);
             let setACopy = new Set(["JavaScript", "HTML", "CSS"]);
             "#}),
-        // Используем копию вместо того же объекта, чтобы избежать конфликта заимствований
+        // We use a copy instead of the same object to avoid borrowing conflicts.
         TestAction::assert_with_op("setA.symmetricDifference(setACopy)", |v, _| {
             v.display().to_string() == "Set(0)"
         }),
     ]);
```

Comment:
I think the comment and the test is wrong, we SHOULD test that using the same set doesn't cause borrowing errors. Testing with a copy would just side step any possible issues.

Reply:
Correct. I just translated the Russian comment to English. The test is wrong.

Test262 already showed the error and I'm correcting it. I'll address this test once all test262 pass.